taccsite_cms/templates/assets_custom.html

```diff
@@ -12,8 +12,8 @@
 
 
 <!-- Custom Site Assets: Favicon. -->
-{% with settings.FAVICON as favicon %}
-<link rel="icon" href="{% static favicon.img_file_src %}" type="image/x-icon" />
+{% with settings.TACC_FAVICON as favicon %}
+<link rel="icon" href="{% if favicon.is_remote %}{{ favicon.img_file_src }}{% else %}{% static favicon.img_file_src %}{% endif %}" type="image/x-icon" />
```

Comment:
Would you be able to make this change for Core-Portal as well?

Reply:
1 and 2: There is the risk of file collision if both portal and cms use the same static directory. Likelihood does seem low. Is there an advantage to both using the same static url?
3: Can you expand on this? I'm not sure what you're referring to exactly

Reply:
**1. & 2.** I can't think of one.
**3.** Unrelated topic I was forcing into the conversation through logic I did not expose. Please never mind for this topic.

**Yes, I can do it.**

On Portal, would the value `/static/...` (to load from Core-CMS) use `is_remote = TRUE`, because it's not using Core-Portal's `{% static %}`?

Reply:
> On Portal, would the value /static/... (to load from Core-CMS) use is_remote = TRUE, because it's not using Core-Portal's {% static %}?

Ah I see, the portal already kinda handles this in that it doesn't default to using `% static %`. Think it's still needed?

Reply:
_I forgot about this. Back now. And reading it without the flu fogging my brain._

I think Portal does not need `{% static %}` option for favicon. It works for remote and CMS assets with its code as is. _To update Portal would be for consistency. A welcome change, but a rabbit hole. The divergence occurred and is great now._

<details><summary>Details (for those who didn't read the thread)</summary>

Adding `{% static %}` and `is_remote` option to Portal adds complexity. The Portal already assumes favicon is not a "local" asset, thus assumes it will be given a non-Portal-static-asset URL. If existing hardcoded `/static/` values concern us, we can add something like `CMS_STATIC_DIR` in default settings.

https://github.com/TACC/Core-Portal/blob/v3.5.0/server/portal/templates/base.html#L11-L13

```py
    {% if settings.PORTAL_ICON_FILENAME %}
    <link rel="icon" href="{{ settings.PORTAL_ICON_FILENAME }}">
    {% else %}
```

</details>

Reply:
@taoteg will test this theory on DigitalRocks. 😀